packages/babel-plugin/src/css-prop/__tests__/object-literal.test.tsx

```diff
@@ -439,7 +442,7 @@ describe('css prop object literal', () => {
       `);
 
     expect(actual).toInclude(`{color:blue}`);
-    expect(actual).toInclude(`{border:1px solid blue}`);
+    expect(actual).toInclude(`{border:1px blue solid}`);
```

Comment:
hehe..ordered 😛 What I was saying. Both works.

Reply:
you're right 😄 it orders it so even if you define it both ways - it ends up being a single declaration instead of multiple